examples/spark2/src/main/scala/org/apache/carbondata/examples/ConcurrencyTest.scala

```diff
@@ -18,44 +18,60 @@
 package org.apache.carbondata.examples
 
 import java.io.File
+import java.text.SimpleDateFormat
 import java.util
+import java.util.Date
 import java.util.concurrent.{Callable, Executors, Future, TimeUnit}
 
 import scala.util.Random
 
 import org.apache.spark.sql.{DataFrame, Row, SaveMode, SparkSession}
 import org.apache.spark.sql.types._
 
-import org.apache.carbondata.core.constants.CarbonCommonConstants
+import org.apache.carbondata.core.constants.{CarbonCommonConstants, CarbonVersionConstants}
 import org.apache.carbondata.core.util.{CarbonProperties, CarbonUtil}
 
 // scalastyle:off println
 object ConcurrencyTest {
```

Comment:
please make a benchmark package and move it there, rename the class to `ConcurrencyBenchmark`

Reply:
ok, done